### DISCUSSION TITLE
Please make browsing albums possible

### DISCUSSION THREAD

**Discussion by datobe** [gokadzev/Musify#342] [Ideas]
When I search for an artist, app detects individual songs but would be great to see whole albums as well.

**Comment by datobe** [comment]
უი ქართველი ხარ? როგორ გამიხარდა :დ

**Comment by gokadzev** [comment]
> When I search for an artist, app detects individual songs but would be great to see whole albums as well.

Unfortunately, the YouTube API doesn't support that, so I'm unable to do it. I manually add already existing albums (of course, I won't be able to add every album that exists in the world, but still...).
